### PR TITLE
release: v1.21.1

### PR DESCRIPTION
**Pull request by iliapolo**
See CHANGELOG

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by iliapolo** [comment]
Ran the integration tests locally - passed 👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.21.1](https://github.com/aws/aws-cdk/compare/v1.21.0...v1.21.1) (2020-01-16)
+
+### Bug Fixes
+
+* **ecr-assets:** fix assembly creation to not include absolute paths to docker files ([#5836](https://github.com/aws/aws-cdk/pull/5836)) ([6bc8ecc](https://github.com/aws/aws-cdk/pull/5833/commits/6bc8eccdba21a10944c81ababe07df37c07481d0)), fixes ([#5807](https://github.com/aws/aws-cdk/issues/5807))
+* **cli:** revert cdk execution from inner directories, due to an issue which will be fixed in a subsequent version ([#5836](https://github.com/aws/aws-cdk/pull/5836)) ([da9c626](https://github.com/aws/aws-cdk/pull/5833/commits/da9c6262fd262a5cfd8c150c2c165e46ecbc95b1))
```
Comment (RomainMuller) [inline CHANGELOG.md]: I feel this should focus more on what the problem was and less on the process used to retract / restore it...

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.21.1](https://github.com/aws/aws-cdk/compare/v1.21.0...v1.21.1) (2020-01-16)
+
+### Bug Fixes
+
+* **ecr-assets:** fix assembly creation to not include absolute paths to docker files ([#5836](https://github.com/aws/aws-cdk/pull/5836)) ([6bc8ecc](https://github.com/aws/aws-cdk/pull/5833/commits/6bc8eccdba21a10944c81ababe07df37c07481d0)), fixes ([#5807](https://github.com/aws/aws-cdk/issues/5807))
```
Comment (RomainMuller) [inline CHANGELOG.md]: Use the same format as other change log entries... I don't think the "fixes" is necessary

Comment (iliapolo) [reply CHANGELOG.md]: Its kind of incosistent. Some do have "closes..". I think its valuable to link to the issue itself as it provides the details for the actual problem.

File: CHANGELOG.md (hunk 1 above)
Comment (iliapolo) [reply CHANGELOG.md]: so something like: "disallow running cdk from inner directories..."

Comment (RomainMuller) [reply CHANGELOG.md]: Tell what was broken, very directly. "CDK init fails if there is a CDK.json file in a parent directory". You could add a "reverts <PR that caused the problem>"

File: CHANGELOG.md (hunk 2 above)
Comment (RomainMuller) [reply CHANGELOG.md]: Ohh. Okay then, disregard!

File: CHANGELOG.md (hunk 1 above)
Comment (iliapolo) [reply CHANGELOG.md]: Yeap, makes sense.

**Pull request merged**